### ISSUE TITLE
Code Highlighting for preprocessor code

### ISSUE THREAD

**Issue by Bellamonte** [esteinberg/plantuml4idea#389]
There is no highlighting for PlantUML preprocessor code.
As I'm writing my PlantUML massively using preprocessor functionality, it get's really hard to read the code, if there is no highlighting of variables, processes, functions and so on.
This should definately be improved.
Thanks! :)

**Comment by krasa** [comment]
![image](https://github.com/esteinberg/plantuml4idea/assets/1160875/8bd6a7cc-35f0-4802-9195-3e982ef94cb5)
like that?

**Comment by Bellamonte** [comment]
Yes, this looks pretty good! Looking forward to this being released. :)

**Comment by krasa** [comment]
done.
btw it is a dumb highlighting, it does not really recognize syntax

**Comment by Bellamonte** [comment]
Oh, but it really helps to be able to have some orienation still, even if it is not that sohphisticated...
Thanks!
One thing: Could you also make it work inside @startdef/@enddef, not only @startuml/@enduml? That would be great! :)
